.github/workflows/website_webhook.yml

```diff
@@ -1,67 +1,42 @@
-name: auto-upload announcements to website
+name: Auto-upload Distro announcements to Bazaar
 
 on:
-  issues:
+  discussion:
     types: [pinned]    
 
 jobs:
  sendAnnouncement:
   runs-on: ubuntu-latest
   steps:
-
-  - name: get release/announcement
-    id: rel_annou
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
+  
+  - name: Fetch announcement
+    id: announcement
     run: |
-      echo $ISSUE_CONTEXT > temp_title.txt
-      echo $(cat temp_title.txt)
-      sed -nr 's/.*"title": (.*), "updated.*/\1/p' <temp_title.txt >title.txt
-      echo $(cat title.txt)
-      if [[ $(cat title.txt) == *"Release"* ]]
+      category="${{ github.event.discussion.category.name }}"
+      title="${{ github.event.discussion.title }}"
+      body="${{ github.event.discussion.body }}"
+      
+      echo "::set-output name=title::${title}"
+      echo "::set-output name=summary::${body:0:500}"
+      echo "::set-output name=description::${body}"
+      
+      if [[ "${category}" == "Releases" ]]
       then
-        echo "::set-output name=release::true"
+        echo "::set-output name=is_release::true"
       else
-        echo "::set-output name=release::false"
-      fi
-
-  - name: Get summary
-    id: summary
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp.txt
-      sed -nr 's/.*### Brief description of the announcement(.*)### Detailed context.*/\1/p' <temp.txt >temp5.txt
-      echo "::set-output name=summary::$(cat temp5.txt)"
-      
-  - name: Get title
-    id: title
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp_title.txt
-      sed -nr 's/.*"title": (.*), "updated.*/\1/p' <temp_title.txt >title.txt
-      echo "::set-output name=title::$(cat title.txt)"
-
-  - name: Get description
-    id: description
-    env:
-      ISSUE_CONTEXT: ${{ toJson(github.event.issue) }}
-    run: |
-      echo $ISSUE_CONTEXT > temp.txt
-      sed -nr 's/.*"body": (.*), "closed_at.*/\1/p' <temp.txt >temp5.txt
-      echo "::set-output name=description::$(cat temp5.txt)"  
+        echo "::set-output name=is_release::false"
+      fi      
```

Comment:
We can remove the "release" from the outputs, and instead pass the category directly - we can then process it on the website side. We want to be able to process not only the releases, but also other categories e.g.: summer schools, announcements, etc.